tests/models/test_converter.py

```diff
@@ -372,7 +372,7 @@ def test_converter_convert_with_invalid_page_close_event_raises_an_exception(
             list(result)
 
 
-@settings(suppress_health_check=(HealthCheck.function_scoped_fixture,))
+@settings(deadline=None, suppress_health_check=(HealthCheck.function_scoped_fixture,))
```

Comment:
Why did you remove this ? Useless?

Reply:
I got the following error during the first run in the CI:

```
Unreliable test timings! On an initial run, this test took 308.32ms, which exceeded the deadline of 200.00ms, but on a subsequent run it took 2.27 ms, which did not. If you expect this sort of variability in your test timings, consider turning deadlines off for this test by setting deadline=None.

You can reproduce this example by temporarily adding @reproduce_failure('6.70.2', b'AXicY2BkYAAiKGBkQObBmIzIHHSAQxgTAAAC6QAJ') as a decorator on your test case
============ 1 failed, 775 passed, 55 warnings in 145.33s (0:02:25) ============
```

I suspect the "suspicious" speedup flagged by `hypothesis` is due to caching performed in our `custom_given` fixture. However, I'm not 100% sure; unfortunately, this error only occurs sometimes.